alf/algorithms/actor_critic_algorithm.py

```diff
@@ -13,15 +13,12 @@
 # limitations under the License.
 """Actor critic algorithm."""
 
-import gin.tf
-import tensorflow as tf
-
-from tf_agents.networks.network import Network, DistributionNetwork
-
+import gin
+from alf.networks import ActorNetwork, ValueNetwork
```

Comment:
ActorNetwork should be ActorDistributionNetwork

Reply:
Changed